pilot/pkg/networking/util/util.go

```diff
@@ -891,8 +885,8 @@ func VersionGreaterOrEqual124(proxy *model.Proxy) bool {
 	return proxy.VersionGreaterAndEqual(&model.IstioVersion{Major: 1, Minor: 24, Patch: -1})
 }
 
-func DelimitedStatsPrefix(statPrefix string, proxyVersion *model.IstioVersion) string {
-	if features.EnableDelimitedStatsTagRegex && IsIstioVersionGE123(proxyVersion) {
```

Comment:
Do we want to cleanup 1.23 as well?

Reply:
Checking if something is GE123 is for 1.22 support - it means we behave differently on old 1.22 proxies